app/services/versioned_files_service.rb

```diff
@@ -34,13 +35,15 @@ def update(version:, version_metadata:, cocina:, file_transfers: {})
     end
   end
 
-  # Deletes a version.
-  # @param version [String] the version number
-  def delete(version:)
+  # Deletes all versions.
+  def delete
     VersionedFilesService::Lock.with_lock(@object) do
-      DeleteAction.new(version:, object: @object).call
-      StacksLinkAction.new(version: head_version, object: @object).call
+      @object.version_manifest.versions.sort.reverse_each do |version|
```

Comment:
Why not just change the DeleteAction to do a FileUtils.rm_rf on stacks object path?